test/logstop_test.rb

```diff
@@ -12,13 +12,26 @@ def test_email
     assert_filtered "test-test@example.org"
     assert_filtered "test@example.us"
     assert_filtered "test@example.science"
+    refute_filtered "test@example.org", email: false
```

Comment:
I think we can test this functionality with only a single assertion for each option.